app/components/Article/FeedbackForm/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import './feedbackForm.css'
+
+export interface FeedbackFormProps {
+  /**
+   * Article ID
+   */
+  pageid: string
+  /**
+   * Class name
+   */
+  className?: string
+}
+const FeedbackForm = ({pageid, className = 'feedback-form'}: FeedbackFormProps) => {
+  // to be implemented.
+  console.log(pageid)
+  const [feedbackOptions, setFeedbackOptions] = React.useState([
+    {
+      text: 'Making things up',
+      selected: false,
+      option: 'making_things_up',
+    },
+    {
+      text: 'Being mean',
+      selected: false,
+      option: 'being_mean',
+    },
+    {
+      text: 'Typos',
+      selected: false,
+      option: 'typos',
+    },
+  ])
+  const [enabledSubmit, setEnabledSubmit] = React.useState(false)
+  const selectFeedback = (option: number) => {
+    setFeedbackOptions(
+      feedbackOptions.map((feedback, index) => {
+        if (index === option) {
+          return {...feedback, selected: !feedback.selected}
+        }
+        return {...feedback, selected: false}
+      })
+    )
+    setEnabledSubmit(true)
+  }
+
+  return (
+    <div className={className}>
```

Comment:
why not just make this into a html form? A radio input + button should handle most of this

Reply:
all other comments have been addressed. this one is slightly more css-nasty.